community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/RemoveAcceptanceTest.scala

```diff
@@ -52,4 +52,52 @@ class RemoveAcceptanceTest extends ExecutionEngineFunSuite with QueryStatisticsT
     assertStats(result, labelsRemoved = 2)
     result.toList should equal(List(Map("labels(n)" -> List("L2"))))
   }
+
+  test("remove a single node property") {
+    // GIVEN
+    createLabeledNode(Map("prop" -> 42), "L")
+
+    // WHEN
+    val result = updateWithBothPlanners("MATCH (n) REMOVE n.prop RETURN exists(n.prop) as still_there")
+
+    //THEN
+    assertStats(result, propertiesSet = 1)
```

Comment:
Curiously the stats did not have a propertiesRemoved counter, so as far as they were concerned removal was seen as a set-property. I took this has a hint! :-)